testkit-backend/src/request-handlers.js

```diff
@@ -126,7 +126,7 @@ export function ResultConsume (context, data, wire) {
       wire.writeResponse('Summary', {
         ...summary,
         serverInfo: {
-          agent: summary.server.agent,
+          agent: summary.server.version,
```

Comment:
It not supposed to be this way. Do you have the test case which is failing? 

The PR i've introduced the agent:  https://github.com/neo4j/neo4j-javascript-driver/pull/728/files

Reply:
Said test is here: https://github.com/neo4j-drivers/testkit/pull/128/files#diff-405b119dba153b17b59d29187cebb451ea0d562ea99ee67c201fa3a3a28bae13R118-R129
It's a WIP and I still have local changes. Not sure how well this PR branch currently works.

Reply:
I will try test in my machine to see what's happening.